src/repository/exercises-repository.ts

```diff
@@ -1304,1527 +1282,3122 @@ export const db: Schema = {
         Nós: 'aumentámos',
         'Eles/Elas/Vocēs': 'aumentaram'
       }
-    }
-  ],
-  nouns: [
+    },
     {
-      english: 'city',
-      portuguese: {
-        word: 'cidade',
-        plural: 'cidades',
-        gender: 'feminine'
+      english: 'to fall',
+      infinitive: 'cair',
+      presentSimple: {
+        Eu: 'caio',
+        Tu: 'cais',
+        'Ela/Ele/Você': 'cai',
+        Nós: 'caímos',
+        'Eles/Elas/Vocēs': 'caem'
+      },
+      pastPerfect: {
+        Eu: 'caí',
+        Tu: 'caíste',
+        'Ela/Ele/Você': 'caiu',
+        Nós: 'caímos',
+        'Eles/Elas/Vocēs': 'caíram'
       }
     },
     {
-      english: 'tea',
-      portuguese: {
-        word: 'chá',
-        gender: 'masculine'
+      english: 'to carry',
+      infinitive: 'carregar',
+      presentSimple: {
+        Eu: 'carrego',
+        Tu: 'carregas',
+        'Ela/Ele/Você': 'carrega',
+        Nós: 'carregamos',
+        'Eles/Elas/Vocēs': 'carregam'
+      },
+      pastPerfect: {
+        Eu: 'carreguei',
+        Tu: 'carregaste',
+        'Ela/Ele/Você': 'carregou',
+        Nós: 'carregámos',
+        'Eles/Elas/Vocēs': 'carregaram'
       }
     },
     {
-      english: 'book',
-      portuguese: {
-        word: 'livro',
-        plural: 'livros',
-        gender: 'masculine'
+      english: 'to arrive',
+      infinitive: 'chegar',
+      presentSimple: {
+        Eu: 'chego',
+        Tu: 'chegas',
+        'Ela/Ele/Você': 'chega',
+        Nós: 'chegamos',
+        'Eles/Elas/Vocēs': 'chegam'
+      },
+      pastPerfect: {
+        Eu: 'cheguei',
+        Tu: 'chegaste',
+        'Ela/Ele/Você': 'chegou',
+        Nós: 'chegámos',
+        'Eles/Elas/Vocēs': 'chegaram'
       }
     },
     {
-      english: 'girlfriend',
-      portuguese: {
-        word: 'namorada',
-        plural: 'namoradas',
-        gender: 'feminine'
+      english: 'to focus',
+      infinitive: 'concentrar',
+      presentSimple: {
+        Eu: 'concentro',
+        Tu: 'concentras',
+        'Ela/Ele/Você': 'concentra',
+        Nós: 'concentramos',
+        'Eles/Elas/Vocēs': 'concentram'
+      },
+      pastPerfect: {
+        Eu: 'concentrei',
+        Tu: 'concentraste',
+        'Ela/Ele/Você': 'concentrou',
+        Nós: 'concentrámos',
+        'Eles/Elas/Vocēs': 'concentraram'
       }
     },
     {
-      english: 'time',
-      portuguese: {
-        word: 'tempo',
-        gender: 'masculine'
+      english: 'to build',
+      infinitive: 'construir',
+      presentSimple: {
+        Eu: 'construo',
+        Tu: 'constróis/construis',
+        'Ela/Ele/Você': 'constrói/construi',
+        Nós: 'construímos',
+        'Eles/Elas/Vocēs': 'constroem/construem'
```

Comment:
meh
is correct according to the portuguese dictionary but I never heard this second version
not to complicate it just say 
Eu: 'construo',
         Tu: 'constróis',
         'Ela/Ele/Você': 'constrói',
         Nós: 'construímos',
         'Eles/Elas/Vocēs': 'constroem'